### DISCUSSION TITLE
Automatic copy button for code blocks

### DISCUSSION THREAD

**Discussion by OCram85** [jpanther/congo#86] [Ideas]
Hi folks,

what do you think about adding a copy button in the code blocks like in the [doks](https://github.com/h-enk/doks) theme?

I think it's a really useful feature if you use congo for developer blogs ;)

### ✨ Additional Resources

- [Feature example in docs](https://getdoks.org/docs/extensions/highlight-js/)
- Example from my previous projects:

![image](https://user-images.githubusercontent.com/13993700/150313681-2c749773-6802-4ee3-9920-0dee2028e975.png)

**Comment by jpanther** [comment]
Thanks for the suggestion. This is something that has been on my roadmap since the start of the project but you're the first person to request it so I'll bump it up the priority. It will probably be something that ends up in version 2 (which is not far off).
